main.css

```diff
@@ -81,14 +86,348 @@
   display: flex;
 }
 
+ul.box4 {
+  margin-block-start: 0.5px;
+  margin-block-end: 0.5px;
+  padding-inline-start: 0.5px;
+}
+
 .social-media {
   align-items: center;
   justify-content: center;
-  height: 20px;
+  height: 20.5px;
   padding-left: 24px;
 }
 
 .social-media img {
   height: 19.962478637695312px;
   width: 19.985925674438477px;
 }
+
+li.social-media {
+  display: flex;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 327px;
+  height: 552px;
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 88px 0.5px;
+}
```

Comment:
Please ensure that the main container of the projects has a `display: grid` according to the requirement.